### ISSUE TITLE
Confirm accuracy of station coordinates

### ISSUE THREAD

**Issue by jclgan** [HakaiInstitute/iys-oos#52]
# Confirm accuracy of station coordinates
### For the most part, the locations are tightly-grouped within reason, **with the exception of Station 60**. Also, the trawl dataset has 3 more stations than any of the abiotic datasets, but there are no paired abiotic measurements.

TODO:
[ ] Get survey data sheets from Brian to verify coordinates of Chl/POM stn 60
[ ] Confirm why there is a trawl 62, 63, and 64, but no trawl 61

**Comment by jclgan** [comment]
Brian agreed with the suggestion that Station 60's longitude discrepancy between datasets is likely a typo.

Confirmed that trawl & cast coordinates will be slightly difference.

Need to follow up Chrys on reference station coordinates for trawls

**Comment by timvdstap** [comment]
I've been in touch with Vladimir Radchenko from NPAFC regarding the proposed station coordinates prior to the 2019 GoA Expedition. We figured that by having these coordinates and associating them to the Station-level in the Event Core, we could determine the proximity in which CTD/chlorophyll etc casts were done. However, Vladimir mentioned that _before the survey started, station coordinates were specified in one degree interval of latitude and one-with-half degree of longitude_ (see email thread). As the proposed station coordinates were on an interval we have opted to only include the actual station coordinates, on the _Cast-level_ within the Event Core. [NPAFC - Station Coordinates - GoA Expedtion 2019.pdf](https://github.com/HakaiInstitute/iys-oos/files/4753465/NPAFC.-.Station.Coordinates.-.GoA.Expedtion.2019.pdf)
